### PR TITLE
Finished check_map_content

### PR DESCRIPTION
**Pull request by ArthurBeznik**
If you're done with everything, check this and if good, merge. :)

### CODE REVIEW

File: srcs/parser/check_map_content.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
+#include <parser.h>
+
+static bool check_valid_char(char c)
+{
+	if (c != ' ' && c != '1' && c != '0'
```
Comment (Zangetsu89) [inline srcs/parser/check_map_content.c:6]: Shouldn't we also check for tabs?

File: srcs/parser/check_map_content.c [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		&& c != 'N' && c != 'S' && c != 'W' && c != 'E')
+		return (false);
+	return (true);
+}
+
+bool	check_map_content(char **map_content)
+{
+	int x;
+	int x_size;
```
Comment (Zangetsu89) [inline srcs/parser/check_map_content.c:15]: this is unused, needs to be removed.

File: srcs/parser/check_map_content.c [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	int y;
+	int y_size;
```
Comment (Zangetsu89) [inline srcs/parser/check_map_content.c:17]: same, this is unused, needs to be removed.

**Review by Zangetsu89 — CHANGES_REQUESTED** [review]

File: srcs/parser/check_map_content.c (hunk 1 above)
Comment (ArthurBeznik) [reply srcs/parser/check_map_content.c:6]: very true

**Comment by ArthurBeznik** [comment]
but is this in theory good to avoid conflicts?

File: srcs/parser/check_map_content.c (hunk 3 above)
Comment (ArthurBeznik) [reply srcs/parser/check_map_content.c:17]: inderdaad

File: srcs/parser/check_map_content.c (hunk 1 above)
Comment (ArthurBeznik) [reply srcs/parser/check_map_content.c:6]: "Spaces are a valid part of themap and are up to you to handle. You must be able to parse any kind of map,as long as it respects the rules of the map." they only mention spaces though so no tabs right?

**Comment by Zangetsu89** [comment]
Yeah this is perfect, it won't create any conflicts for sure.

File: srcs/parser/check_map_content.c (hunk 1 above)
Comment (Zangetsu89) [reply srcs/parser/check_map_content.c:6]: Ah true, I thought the spaces before the lines in the map example they gave were tabs. nvm. this is good.

**Review by Zangetsu89 — APPROVED** [review]

**Pull request merged**